pkg/kp/statusstore/podstatus/status.go

```diff
@@ -23,6 +24,10 @@ const (
 	Pending  ServiceState = "pending"
 	Running  ServiceState = "running"
 	Finished ServiceState = "finished"
+
+	PodLaunched PodState = "launched"
```

Comment:
I can't see where these are used in this PR so I must express my concern that the strings are duplicated with the above states. If it's possible for these to appear under the same field then that may confuse debugging.